src/constants.rs

```diff
@@ -96,6 +96,6 @@ pub const UNWRAP_EMOJI: Emoji<'_, '_> = Emoji("🔩 ", "");
 
 pub const MAX_FREEZE_DAYS: u8 = 31;
 
-pub const COMPUTE_UNITS: u32 = 100_000;
+pub const COMPUTE_UNITS: u32 = 400_000;
```

Comment:
Why did you increase the compute units? this should not be required.

Reply:
This is actually a revert to the original https://github.com/metaplex-foundation/sugar/blob/886c6f063b40a0f3504a23711268d11d85d226a8/src/constants.rs#L99

Reply:
missed that - sorry!